install_scripts/helpers.py

```diff
@@ -321,7 +321,9 @@ def base64_encode(data):
 
 def get_docker_deb_pkg_version(docker_version, lsb_dist, dist_version):
     major, minor, patch = map(int, docker_version.split('.'))
-    if major == 1:
+    if major == 18:
+        return '' # unused
```

Comment:
why unused?

Reply:
It appears to install the latest from docker-ce channel, which is currently 18.09. I have another PR open that needs to install 18.06 for K8s 1.13 (since Docker 18.09 is not yet tested with K8s). You can go ahead and merge this and then I'll update my PR to install fixed versions in 18.X.

Reply:
the new script supports version pinning